config/Readme.md

```diff
@@ -0,0 +1,40 @@
+# IMAGE Config
+
+This Folder (Config) should contain the .env files required for IMAGE server preprocessors to run precisely.
+
+## Environment Files
+
+### maps.env
+This env file is required by the following preprocessors:
+* [autor preprocessor](https://github.com/Shared-Reality-Lab/IMAGE-server/tree/main/preprocessors/autour)
+* [openstreetmap preprocessor](https://github.com/Shared-Reality-Lab/IMAGE-server/tree/main/preprocessors/openstreetmap)
+
+Following is the sample format of maps.env file:
```

Comment:
Please explain what the key is (e.g., how someone would obtain a valid API key). This can just be a link.